
int8 robot_id
float32 timestamp
Distance[] ranges